WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -280,8 +281,9 @@ public class ReaderDetailViewController: UIViewController, UIViewControllerResto
             postID.unsignedIntegerValue,
             forSite: siteID.unsignedIntegerValue,
             success: {[weak self] (post:ReaderPost!) in
-                self?.post = post
                 WPNoResultsView.removeFromView(self?.view)
+                self?.textView.alpha = 1.0
```

Comment:
Nipticky: we _could_ use *WPAlphaFull* / *WPAlphaZero*, but since those are not really Swift Friendly... i'm not sure about it. Sounds like a Struct with a bunch of static constants right?

No big deal, just thinking aloud!